webapp/src/main/java/com/box/l10n/mojito/service/tm/TMService.java

```diff
@@ -709,7 +710,11 @@ public String generateLocalized(
 
         driver.addStep(new RawDocumentToFilterEventsStep());
         driver.addStep(new CheckForDoNotTranslateStep());
-        driver.addStep(new TranslateStep(asset, repositoryLocale, InheritanceMode.USE_PARENT));
+        if (bcp47Tag.equals("en-x-psaccent")) {
+            driver.addStep(new PseudoLocalizeStep());
+        } else {
+            driver.addStep(new TranslateStep(asset, repositoryLocale, InheritanceMode.USE_PARENT));
```

Comment:
so if we're doing that here... there is no point of having a different WS, it would work by calling the normal "pull" command and `getLocalizedAssetForContent` WS. 

I think @jeeyi  was more leaning toward an new webservice + command... then we should probably have 2 function in the `TMService`